var/spack/repos/builtin/packages/quantum-espresso/package.py

```diff
@@ -216,6 +219,10 @@ def install(self, spec, prefix):
             options.append(
                 'FFTW_INCLUDE={0}'.format(join_path(env['MKLROOT'],
                                                     'include/fftw')))
+            options.append(
+                'LIBDIRS={0}'.format(join_path(env['MKLROOT'],
+                                     'lib/intel64')))
```

Comment:
@adamjstewart Isn't there away to add `lib/intel64` in an architecture agnostic manner?

Reply:
`self.spec['mkl'].prefix.lib` probably should do; will check that

Reply:
this change even can be omitted completely :-)